hybrid/compartment_examples/inter_comp_call/secure-redirect_clr/main.c

```diff
@@ -0,0 +1,169 @@
+/* This example attempts to set a `clr` which would jump to a malicious
```

Comment:
Could we highlight the line(s) where the error will happen?

Reply:
I've added some reference to specific lines for both examples [1] [2]. Let me know if further clarification would be useful.

[1] https://github.com/capablevms/cheri-examples/pull/57/commits/c261cafc26a475cf8df2770c5afecb1b26f95093
[2] https://github.com/capablevms/cheri-examples/pull/57/commits/f62251051773440a4427a724ac22944980644c2e